Foundation/HTTPCookieStorage.swift

```diff
@@ -200,6 +211,9 @@ open class HTTPCookieStorage: NSObject {
     }
 
     private func updatePersistentStore() {
+        guard !isEphemeral else { return }
```

Comment:
looks like we could combine these guard statements here.

Reply:
@manojmahapatra I'd like to separate concerns. However, I think an `if statement` made more sense here. It makes the code more readable and simple. Thanks!
```
if isEphemeral { return }
```